packages/template/package.json

```diff
@@ -27,7 +27,7 @@
     "configuration-schema.json"
   ],
   "dependencies": {
-    "@openfn/language-common": "^1.8.1"
+    "@openfn/language-common": "^1.12.0"
```

Comment:
TODO!! Template needs its version bumping, I guess?

Or maybe we should forever lock it at 1.0.0?